Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/TextInputAutocapitalizationModifier.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  TextInputAutocapitalizationModifier.swift
+//  LiveViewNative
+//
+//  Created by Dylan.Ginsburg on 4/18/23.
+//
+
+import SwiftUI
+
+/// Sets how often the shift key in the keyboard is automatically enabled.
+///
+/// ```html
+/// <TextField modifiers={autocapitalization(@native, type: :words)}>
+///     Full name
+/// </TextField>
+/// ```
+///
+/// ## Arguments
+/// * ``autocapitalization``
+#if swift(>=5.8)
+@_documentation(visibility: public)
+#endif
+@available(iOS 16.0, watchOS 9.0, *)
+struct TextInputAutocapitalizationModifier: ViewModifier, Decodable, Equatable {
+    /// One of the capitalizing behaviors defined in the `TextInputAutocapitalization` struct or nil.
```

Comment:
Can you list the possible values here?
```suggestion
    /// One of the capitalizing behaviors defined in the `TextInputAutocapitalization` struct or nil.
    ///
    /// Possible values:
    /// * `characters`
    /// * `sentences`
    /// * `words`
    /// * `never`
```

Reply:
Of course I can and will do this moving forward. I was modeling my documentation from existing modifiers. If we want the documentation to be consistent about, well... enumerating the enums then there are some others to fix.

Reply:
Yeah, I think its only important if we don't extend the type to be `Decodable` (in which case the cases should be listed on the extension's docs).